pkg/crc/machine/stop.go

```diff
@@ -9,6 +9,9 @@ import (
 )
 
 func (client *client) Stop() (state.State, error) {
+	if running, _ := client.IsRunning(); !running {
+		return state.Error, errors.New("Cluster is already stopped")
```

Comment:
@anjannath With https://github.com/code-ready/crc/commit/e2a76bbba5653a2c9046efce6753f075057cda06 it should be changed to `Instance is already stopped` . This also need rebase with dev branch.